src/utils/ddb/ddb_commands.c

```diff
@@ -1451,3 +1451,65 @@ ddb_run_dtx_stat(struct ddb_ctx *ctx, struct dtx_stat_options *opt)
 
 	return rc;
 }
+
+int
+ddb_run_dtx_aggr(struct ddb_ctx *ctx, struct dtx_aggr_options *opt)
+{
+	struct dv_indexed_tree_path itp = {0};
+	daos_handle_t               coh = {0};
+	uint64_t                   *epoch;
+	int                         rc;
+
+	if (!ctx->dc_write_mode) {
+		ddb_error(ctx, error_msg_write_mode_only);
+		return -DER_INVAL;
+	}
+
+	rc = init_path(ctx, opt->path, &itp);
+	if (!SUCCESS(rc))
+		goto done;
+
+	if (!itp_has_cont(&itp)) {
+		rc = -DER_INVAL;
+		goto done;
+	}
+
+	rc = dv_cont_open(ctx->dc_poh, itp_cont(&itp), &coh);
+	if (!SUCCESS(rc))
+		goto done;
+
+	switch (opt->format) {
+	case DDB_DTX_AGGR_NOW:
+		epoch = NULL;
```

Comment:
If neither `epoch` nor `date` is specified, then it will aggregate all committed DTX entries by default, right? that can work, but some dangerous. Because some relative new committed DTX entries maybe used by DTX resync/refresh logic after engine start.

So I would suggest that:
1. If neither `epoch` nor `date` is specified, then fail out. Or
2. We define an default `epoch` internally, that maybe about 3 minutes older than the max epoch in the committed DTX table, and then all the committed DTX entries with older epoch can be aggregated by default.

How do you think?

Reply:
I am fine for the first option, that can be enhanced if we want to support the 2nd option in the future.